Python/Tests/ReplWindowUITestsRunner/ReplWindowAdvancedUITests.cs

```diff
@@ -471,6 +471,28 @@ public abstract class ReplWindowAdvancedUITests {
         }
 
         #endregion
+
+        #region Advanced Launch Configuration Tests
+
+        [TestMethod, Priority(2)]
```

Comment:
Are these already flaky? New tests should be Priority(0), and if they aren't reliable then invest the time to make them reliable.

Reply:
Yes, all REPL UI tests are flaky due to #2701 and #3099.

Reply:
Okay